### DISCUSSION TITLE
Firmware only giving me manual bed level option

### DISCUSSION THREAD

**Discussion by BulldawgDesigns** [Jyers/Marlin#695] [Q&A] [answered]
I downloaded E3V2-BLTouch-5x5-HS-v4.2.2 and the only options it gives me are for manual bed leveling when I flash it to my system.
Is there something wrong with the download?

**Comment by Jyers** [comment, marked as answer]
Just to confirm, if you go to the leveling menu and select "Create new mesh" it goes to a manual leveling screen and does not start leveling automatically?
